e2e/go.mod

```diff
@@ -215,3 +234,9 @@ replace (
 
 // uncomment to use the local version of ibc-go, you will need to run `go mod tidy` in e2e directory.
 replace github.com/cosmos/ibc-go/v6 => ../
+
+replace (
+	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+	// ibctest 0.47 @ https://github.com/strangelove-ventures/ibctest/pull/337
+	github.com/strangelove-ventures/ibctest/v6 => github.com/julienrbrt/ibctest/v6 v6.0.0-20221212124331-0930299cd224
```

Comment:
why is this replace here?

Reply:
We need a commit from ibctest which is updated to v0.47. Currently those changes are [unmerged](https://github.com/strangelove-ventures/ibctest/pull/337)